src/DurableSDK/OrchestrationInvoker.cs

```diff
@@ -72,7 +112,12 @@ public Hashtable Invoke(OrchestrationBindingInfo orchestrationBindingInfo, IPowe
             object customStatus)
         {
             var orchestrationMessage = new OrchestrationMessage(isDone, actions, output, customStatus);
-            return new Hashtable { { AzFunctionInfo.DollarReturn, orchestrationMessage } };
+            return new Hashtable { { "$return", orchestrationMessage } };
```

Comment:
note to self: looks like we should convert `"$return"` back to `AzFunctionInfo.DollarReturn`